consensus/core/src/block.rs

```diff
@@ -208,7 +217,13 @@ impl Hash for BlockRef {
     }
 }
 
-/// Hash of a block, covers all fields except signature.
+/// Hash of a block, covers all fields except its signature.
```

Comment:
What is the benefit of having multiple digests? When would you use one over the other? Atleast in the usage below it seems to be between SignedBlock vs VerifiedBlock.

Reply:
`InnerBlockDigest` should never be used outside of signing and verifications. Will update the comments.